.github/workflows/test-ios-simulator.yaml

```diff
@@ -70,62 +72,57 @@ jobs:
             --exclude integration_test/service_cellular_test.dart \
             --exclude integration_test/service_wifi_test.dart \
             --exclude integration_test/swipe_test.dart \
-            --exclude integration_test/webview_leancode_test.dart \
             --exclude integration_test/webview_login_test.dart \
-            --exclude integration_test/webview_stackoverflow_test.dart || EXIT_CODE=$?
+            --exclude integration_test/webview_stackoverflow_test.dart || TESTS_EXIT_CODE=$?
 
-          kill $recordingpid
-          exit $EXIT_CODE
+          kill -SIGINT $recordingpid
+          kill -SIGINT $logpid
+          echo "TESTS_EXIT_CODE=$TESTS_EXIT_CODE" >> $GITHUB_ENV
+          exit $TESTS_EXIT_CODE
 
+      - name: Find xcresult path
+        if: success() || failure()
+        run: echo "XCRESULT_PATH=$(find ~/Library/Developer/Xcode/DerivedData -name "*.xcresult" -type d)" >> $GITHUB_ENV
+      
       - name: Publish test report to summary
         if: success() || failure()
         uses: kishikawakatsumi/xcresulttool@v1
         with:
-          title: "Patrol tests"
+          title: "Patrol tests on ${{ matrix.device }}"
           upload-bundles: "never"
           path: |
             ${{ env.XCRESULT_PATH }}
-
-      - name: Find xcresult path
-        if: success() || failure()
-        run: echo "XCRESULT_PATH=$(find ~/Library/Developer/Xcode/DerivedData -name "*.xcresult" -type d)" >> $GITHUB_ENV
-
-      - name: Upload XCRESULT test report to artifacts
+     
+      - name: Upload XCRESULT test result to artifacts
         if: success() || failure()
         uses: actions/upload-artifact@v3
         with:
-          name: XCRESULT test report from ${{ matrix.device }}
-          path: |
-            ${{ env.XCRESULT_PATH }}
+          name: Test result from ${{ matrix.device }}.xcresult
+          path: ${{ env.XCRESULT_PATH }}
       
       - name: Upload simulator logs to artifacts
         if: success() || failure()
         uses: actions/upload-artifact@v3
         with:
           name: Logs from ${{ matrix.device }}
-          path: ~/Library/Logs/CoreSimulator/${{ steps.start_simulator.outputs.udid }}/system.log
+          path: |
+            ${{ github.workspace }}/packages/patrol/example/all_simulator_logs.txt
 
       - name: Upload captured video to artifacts
         if: success() || failure()
         uses: actions/upload-artifact@v3
         with:
-          name: Captured video from ${{ matrix.device }}
+          name: Captured video from ${{ matrix.device }}.mp4
```

Comment:
Is this why the video was broken?xD

Reply:
XD, noo. I was too brutal while killing recording. The solution is to send SIGINT.

Reply:
XD